.github/config/uncovered_files.csv

```diff
@@ -501,6 +501,7 @@ main/relation/create_view_relation.cpp	7
 main/relation/cross_product_relation.cpp	2
 main/relation/read_json_relation.cpp	10
 main/relation/write_parquet_relation.cpp	22
+main/relation/aggregate_relation.cpp	14
```

Comment:
because the added lines are only tested by python code and not the default test runner right?

Reply:
Yea sadly, I tried at some point to add the python tests to the check_coverage.sh script, but locally it wouldn't budge.
Though without it it also fails locally for me, so it might be worth to just try adding it and running CI with it